ballerina/resiliency_load_balance_client.bal

```diff
@@ -424,9 +424,11 @@ public client isolated class LoadBalanceClient {
                 // When performing passthrough scenarios using Load Balance connector,
                 // message needs to be built before trying out the load balance endpoints to keep the request message
                 // to load balance the messages in case of failure.
-                byte[]|error binaryPayload = loadBalancerInRequest.getBinaryPayload();
-                if binaryPayload is error {
-                    log:printDebug("Error building payload for request load balance: " + binaryPayload.message());
+                if !loadBalancerInRequest.hasMsgDataSource() {
+                    byte[]|error binaryPayload = loadBalancerInRequest.getBinaryPayload();
+                    if binaryPayload is error {
```

Comment:
Do we only log this?

Reply:
Yes, this becoming an error is very rare and unexpected scenario. Thats why we just log and continue.

Reply:
Yeah, we can just continue with logging.